test/cpp/end2end/sdk_authz_end2end_test.cc

```diff
@@ -34,19 +35,53 @@ namespace grpc {
 namespace testing {
 namespace {
 
+constexpr char kCaCertPath[] = "src/core/tsi/test_creds/ca.pem";
+constexpr char kServerCertPath[] = "src/core/tsi/test_creds/server1.pem";
+constexpr char kServerKeyPath[] = "src/core/tsi/test_creds/server1.key";
+constexpr char kClientCertPath[] = "src/core/tsi/test_creds/client.pem";
+constexpr char kClientKeyPath[] = "src/core/tsi/test_creds/client.key";
+
 constexpr char kMessage[] = "Hello";
 
+std::string ReadFile(const char* file_path) {
+  grpc_slice slice;
+  GPR_ASSERT(
+      GRPC_LOG_IF_ERROR("load_file", grpc_load_file(file_path, 0, &slice)));
+  std::string file_contents(grpc_core::StringViewFromSlice(slice));
+  grpc_slice_unref(slice);
+  return file_contents;
+}
+
 class SdkAuthzEnd2EndTest : public ::testing::Test {
  protected:
   SdkAuthzEnd2EndTest()
       : server_address_(
-            absl::StrCat("localhost:", grpc_pick_unused_port_or_die())),
-        server_creds_(
-            std::shared_ptr<ServerCredentials>(new SecureServerCredentials(
-                grpc_fake_transport_security_server_credentials_create()))),
-        channel_creds_(
-            std::shared_ptr<ChannelCredentials>(new SecureChannelCredentials(
-                grpc_fake_transport_security_credentials_create()))) {}
+            absl::StrCat("localhost:", grpc_pick_unused_port_or_die())) {
+    std::string root_cert = ReadFile(kCaCertPath);
+    std::string identity_cert = ReadFile(kServerCertPath);
+    std::string private_key = ReadFile(kServerKeyPath);
+    std::vector<experimental::IdentityKeyCertPair>
+        server_identity_key_cert_pairs = {{private_key, identity_cert}};
+    grpc::experimental::TlsServerCredentialsOptions server_options(
+        std::make_shared<grpc::experimental::StaticDataCertificateProvider>(
+            root_cert, server_identity_key_cert_pairs));
+    server_options.watch_root_certs();
+    server_options.watch_identity_key_cert_pairs();
+    server_options.set_cert_request_type(
+        GRPC_SSL_REQUEST_CLIENT_CERTIFICATE_AND_VERIFY);
+    server_creds_ = grpc::experimental::TlsServerCredentials(server_options);
+
```

Comment:
Please remove blank line.

Reply:
Doesn't look like this was done.

Reply:
Sorry. missed it. Removed the line in the last commit.